cosmovisor/logger.go

```diff
@@ -13,3 +13,11 @@ func SetupLogging() {
 	output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: time.Kitchen}
 	Logger = zerolog.New(output).With().Str("module", "cosmovisor").Timestamp().Logger()
 }
+
+func DisableLogger() {
```

Comment:
nit: add godocs for these two functions. We should also eventually remove the use of a global logger in favor of a logger instance.

Reply:
I totally agree, I had a commit like that but it required much changes, I will bring that back :)